### PR TITLE
Removing exposed ports in dev compose

### PR DESCRIPTION
**Pull request by astitva1905**

### CODE REVIEW

File: docker/compose.dev.yml [hunk 1]
```diff
@@ -36,7 +34,7 @@ services:
     image: traefik:v2.10
     container_name: traefik-gateway
     ports:
-      - "9000:80"
+      - "80:80"
     volumes:
       - "../servers/config/gateway/traefik.yml:/etc/traefik/traefik.yml"
```
Comment (prasadtalasila) [inline docker/compose.dev.yml:39]: The next three lines correspond to traefik using file-based config. Please remove them. It needs to be replaced with pure docker-style config.

**Review by prasadtalasila — COMMENTED** [review]
In addition to the suggested changes:

1. Update README file. Remove `deploy/config/gateway/auth` line
2. Update `compose.local.yml`

File: docker/compose.dev.yml [hunk 2]
```diff
@@ -36,7 +34,7 @@ services:
     image: traefik:v2.10
     container_name: traefik-gateway
     ports:
-      - "9000:80"
+      - "80:80"
```
[the remark this answers is not in the document]
Comment (prasadtalasila) [reply docker/compose.dev.yml:37]: @astitva1905 please revert the suggestion. Do keep the port 80. Sorry for creating the confusion.

**Comment by astitva1905** [comment]
Shifts relevant changes to a new PR that includes the new structure for docker directory

**Pull request closed without merging**